tree/30_generic_methods/audit_from_osquery.cf

```diff
@@ -61,6 +61,8 @@
 # @parameter_constraint comparator "select" : [ "", "=", "!=", "~" ]
 # @parameter value           The expected value
 # 
+# @parameter_name query condition_prefix
```

Comment:
I don't understand this one

Reply:
Well spotted, I deleted this line, fixup coming